<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministratorClass</class>
 <widget class="QMainWindow" name="AdministratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QListWidget" name="dogsList"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="breedLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Breed:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>breedEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="breedEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name:</string>
            </property>
            <property name="buddy">
             <cstring>nameEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="ageLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Age:</string>
            </property>
            <property name="buddy">
             <cstring>ageEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ageEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="weightLagel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Weight:</string>
            </property>
            <property name="buddy">
             <cstring>weightEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="weightEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="photoLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Photo link:</string>
            </property>
            <property name="buddy">
             <cstring>photoEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="photoEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addDogButton">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeDogButton">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="updateDogButton">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="filterDogButton">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_6">
          <item row="0" column="1">
           <widget class="QLineEdit" name="weightFilterEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="weightFilterLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Filter by weight:</string>
            </property>
            <property name="buddy">
             <cstring>weightFilterEdit</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="filterAgeLabel">
            <property name="font">
             <font>
              <family>Verdana</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Filter by age:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="filterAgeLineEdit">
            <property name="font">
             <font>
              <family>Verdana</family>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Lab1214.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addDogButton</sender>
   <signal>clicked()</signal>
   <receiver>AdministratorClass</receiver>
   <slot>addButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeDogButton</sender>
   <signal>clicked()</signal>
   <receiver>AdministratorClass</receiver>
   <slot>removeButtonHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterAgeLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AdministratorClass</receiver>
   <slot>filterAgeDogsHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterAgeLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AdministratorClass</receiver>
   <slot>filterAgeDogsHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>514</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateDogButton</sender>
   <signal>clicked()</signal>
   <receiver>AdministratorClass</receiver>
   <slot>uptadeButtonHandel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>683</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>addButtonSignal()</signal>
  <signal>removeButtonSignal()</signal>
  <slot>addButtonHandler()</slot>
  <slot>removeButtonHandler()</slot>
  <slot>filterAgeDogsHandler()</slot>
  <slot>uptadeButtonHandel()</slot>
 </slots>
</ui>
